lib/merchant.rb

```diff
@@ -1,3 +1,5 @@
+require 'date'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.